doc/content/verification_and_validation/val-2d.md

```diff
@@ -0,0 +1,167 @@
+
+\begin{equation} \label{eq:diffusivity}
+D = D_{0} \exp \left( - \frac{E_{D}}{k_b T} \right),
+\end{equation}
+
+where $E_{D}$ is the diffusion activation energy, $k_b$ is the Boltzmann’s constant, $T$ is the temperature, and $D_{0}$ is the maximum diffusivity coefficient. TMAP7 ([!citep](ambrosek2008verification)) assigns two different value, $D_{0,l}$ and $D_{0,r}$, for implantation zone ($x < 15 \times 10 ^ {-9}$ m) and other zone ($x > 15 \times 10 ^ {-9}$ m).
+
+$\alpha_t^i$ and $\alpha_r^i$ are defined as:
+
+\begin{equation} \label{eq:trapping}
+\alpha_t^i = \alpha_{t0}^i \exp(-\epsilon_t^i / T),
+\end{equation}
+
+and
+
+\begin{equation} \label{eq:release}
+\alpha_r^i = \alpha_{r0}^i \exp(-\epsilon_r^i / T),
+\end{equation}
+
+where $\alpha_{t0}^i$ and $\alpha_{r0}^i$ are pre-exponential factors of trapping and release rate coefficients, respectively, $\epsilon_t^i$ and $\epsilon_r^i$ are the trapping and detrapping energies, respectively.
+
+The thermal diffusion after 5000s is governing by:
+
+\begin{equation} \label{eq:thermal}
+\rho C_P \frac{d T}{d t} = \nabla D_T \nabla T,
+\end{equation}
+
+where $\rho$ is the density of Tungsten, $C_P$ is the specific heat, $D_T$ is the thermal conductivity.
+
+All the model parameters are taken from [!cite](hino1998hydrogen,haasz1999effect,anderl1992deuterium,frauenfelder1969solution,ambrosek2008verification) and listed in [val-2d_set_up_values].
+
+## Model Description
+
+In this case, TMAP8 simulates a one-dimensional domain to represent the tritium implantation, diffusion, and trapping and detrapping in a raising thermal field. Note that this case can easily be extended to a two- or three-dimensional case.
+
+The source term in the model is described as a normal distribution in [eq:normal_distribution]:
+
+\begin{equation} \label{eq:normal_distribution}
+S = F \frac{1}{\sigma \sqrt{2 \pi}} \exp \left( - \frac{(x - \mu )^2}{2 \sigma^2} \right),
+\end{equation}
+
+where $F$ is the implantation flux provided in [val-2d_set_up_values], $\sigma = 3 \times 10^{-9}$ m is the characteristic width of the normal distribution, and $\mu = 4.6 \times 10^{-9}$ m is the depth of the normal distribution from the upstream side.
+
+The pressures on the upstream and downstream sides are close to vacuum pressures, and have less impact on tritium desorption. Thus, TMAP8 ignores the pressure in these enclosures. Also, because both surfdep and Dirichlet boundary condition have similar desorption performance in the near vacuum environment, we use Dirichlet boundary condition to improve the modal efficiency. The concentrations on upstream and downstream sides are defined as:
```

Comment:
```suggestion
The pressures on the upstream and downstream sides are close to vacuum pressures, and do not have a significant impact on tritium desorption. Thus, TMAP8 ignores the pressure in these enclosures for simplification. Also, because both surfdep and Dirichlet boundary conditions have similar desorption performance in the near vacuum environment, we use Dirichlet boundary condition to improve the model efficiency. The concentrations on upstream and downstream sides are defined as:
```